bin/cron/teacher_applications_to_gdrive

```diff
@@ -25,6 +25,8 @@ def get_rows
     if app.sanitize_form_data_hash[:principal_underrepresented_minority_percent]
       urm_percent = app.sanitize_form_data_hash[:principal_underrepresented_minority_percent].to_f
     end
+    accepted_statuses = %w(accepted_no_cost_registration accepted_notified_by_partner registration_sent paid accepted_not_notified)
+    accepted = accepted_statuses.include?(app.status) ? 1 : 0
```

Comment:
are we using numbers here so they can easily count the total number of acceptances?

Reply:
0/1 was the request from programs, they are currently semi-manually computing this column and putting 0/1 in, so I would guess it's for counting?